### PR TITLE
Add pip instructions

### PR DESCRIPTION
**Pull request by milancurcic**
Expand README a bit and add instructions for pip.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+
+### Build fpm-docs
+
 Build the documentation by invoking
 
 ```
 make html
 ```
+
+The website will be built in \_build/html.
```
Comment (awvwgk) [inline README.md:44]:
````suggestion
The website will be built in `_build/html` and can be previewed by opening the page with a browser (*e.g.* firefox, chromium or similar):

```
firefox file://$PWD/_build/html/index.html
```

By default all languages will be build, to limit the build to a single language subtree, *i.e.* English, use

```
make html LANGUAGES=en
```
````

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 /_*
 *.mo
+
+venv
```
Comment (awvwgk) [inline .gitignore:4]: only ignore `venv/` directories? Maybe limited to the project root:
```suggestion
/venv/
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,44 @@
...
+```
+git clone https://github.com/fortran-lang/fpm-docs
+cd fpm-docs
+```
+
+### Install dependencies
+
+The documentation pages are created using [sphinx](https://www.sphinx-doc.org).
+You first need to install the required dependencies using conda or pip.
+
```
Comment (awvwgk) [inline README.md:19]:
```suggestion
This project depends on

- [sphinx-build](https://www.sphinx-doc.org) for building the pages
- [sphinx-intl](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html) for translations
- [ablog](https://ablog.readthedocs.io/en/latest/) for the news section and posts
- [sphinx-book-theme](https://sphinx-book-theme.readthedocs.io/en/latest/) for the page theme
- [myst-parser](https://myst-parser.readthedocs.io/en/latest/) for markdown support
- [sphinx-design](https://sphinx-design.readthedocs.io/en/latest/) for the bootstrap building blocks

```

**Review by awvwgk — COMMENTED** [review]
Looks good, I left a few suggestions

File: README.md [hunk 4]
```diff
@@ -3,17 +3,61 @@
...
 make html
 ```
+
+The website will be built in `_build/html` and can be previewed by opening the page with a browser (*e.g.* firefox, chromium or similar):
+
+```
+firefox file://$PWD/_build/html/index.html
+```
+
+By default all languages will be build, to limit the build to a single language subtree, *i.e.* English, use
```
Comment (ivan-pi) [inline README.md:59]:
```suggestion
By default all languages will be built, to limit the build to a single language subtree, *i.e.* English, use
```

Comment (milancurcic) [reply README.md:59]: Thanks, good catch

File: README.md [hunk 5]
```diff
@@ -3,17 +3,61 @@
...
 make html
 ```
+
+The website will be built in `_build/html` and can be previewed by opening the page with a browser (*e.g.* firefox, chromium or similar):
+
+```
+firefox file://$PWD/_build/html/index.html
+```
+
+By default all languages will be built, to limit the build to a single language subtree, *i.e.* English, use
```
Comment (milancurcic) [inline README.md:59]: This is two sentences in one, let's split it.

```suggestion
By default all languages will be built.
To limit the build to a single language subtree, *i.e.* English, use
```

**Pull request merged**